docs/pyproject.toml

```diff
@@ -6,7 +6,7 @@ authors = ["ScyllaDB Contributors"]
 
 [tool.poetry.dependencies]
 python = "^3.7"
-pyyaml = "6.0"
+pyyaml = "6.0.1"
```

Comment:
might want to note the reason why we need to bump up `pyyaml` . even 

> we want to use the latest stable release of `pyyaml` 

is a good reason than no reasons =)

Reply:
Added to the pull request description - if not I get an error on Mac.